### PR TITLE
compress error msg for run_all_miracl

### PR DESCRIPTION
**Pull request by crystina-z**
For issue #1325

### CODE REVIEW

**Review by lintool — APPROVED** [review]

**Review by lintool — CHANGES_REQUESTED** [review]
Hi @crystina-z I just realized - this won't work for the dev bindings that we have.

Right now, we can do `--topics miracl-v1.0-ar-dev` and it'll work.

With your path, I think the code will throw an error?

**Comment by crystina-z** [comment]
right but the `run_all_miracl.py` would only read in `miracl.yaml`, which writes the topic path

**Comment by crystina-z** [comment]
```
condition bm25.ar:                                                              
  - split: train          
    Running: python -m pyserini.search.lucene --language ar --topics miracl-v1.0-ar-train --index miracl-v1.0-ar --output runs/run.miracl.bm25.ar.train.txt --batch 128 --threads 16 --bm25 --hits 100
Skip miracl-v1.0-ar-train.                                                      
  - split: dev            
      nDCG@10: 0.4809 [OK]
      R@100  : 0.8885 [OK]   
```

^^^ current output

File: scripts/repro_matrix/run_all_miracl.py [hunk 1]
```diff
@@ -70,12 +77,17 @@ def print_results(metric, split):
 
                 if not os.path.exists(runfile):
                     print(f'    Running: {cmd}')
-                    os.system(cmd)
+                    rtn = subprocess.run(cmd.split(), capture_output=True)
+                    stderr = rtn.stderr.decode()
+                    topic_fn = extract_topic_fn_from_cmd(cmd)
+                    if f"ValueError: Topic {topic_fn} Not Found" in stderr:
+                        print(f"Skip {topic_fn}.")
```
Comment (lintool) [inline scripts/repro_matrix/run_all_miracl.py:84]:
```suggestion
                        print(f"Skipping {topic_fn}: file not found.")
```

Comment (lintool) [reply scripts/repro_matrix/run_all_miracl.py:84]: Also, change to single quotes?

**Review by lintool — APPROVED** [review]
Address minor nits and go ahead and merge please.

**Pull request merged**